helmfile.d/helmfile-06.init.yaml

```diff
@@ -33,13 +33,12 @@ releases:
     values:
       - ../values/istio-operator/istio-operator-raw.gotmpl
       - snippets/common.gotmpl
-  - name: external-secrets
```

Comment:
Please add removal of this Helm release to the upgrade.yaml file

Reply:
It has just been moved

Reply:
no, no it was moved to different namespace, therefor it is a different helm release

Reply:
Ok

Reply:
But I am on the road for the next three days, so please, if you would want to add that one line to this PR I would be very grateful

Reply:
sure, I will do so